Description: Fix iscsi target path issues.
Author: Chuck Short <zulcss@ubuntu.com>
Fowrded: Not needed
diff -Naurp nova-2011.3.orig//nova/volume/driver.py nova-2011.3//nova/volume/driver.py
--- nova-2011.3.orig//nova/volume/driver.py	2011-10-12 13:33:29.000000000 -0400
+++ nova-2011.3//nova/volume/driver.py	2011-10-12 13:39:51.082109791 -0400
@@ -533,9 +533,14 @@ class ISCSIDriver(VolumeDriver):
 
         self._iscsiadm_update(iscsi_properties, "node.startup", "automatic")
 
-        mount_device = ("/dev/disk/by-path/ip-%s-iscsi-%s-lun-0" %
-                        (iscsi_properties['target_portal'],
-                         iscsi_properties['target_iqn']))
+        if FLAGS.iscsi_helper == 'tgtadm':
+            mount_device = ("/dev/disk/by-path/ip-%s-iscsi-%s-lun-0" %
+                            (iscsi_properties['target_portal'],
+                             iscsi_properties['target_iqn']))
+        else:
+            mount_device = ("/dev/disk/by-path/ip-%s-iscsi-%s-lun-0" %
+                            (iscsi_properties['target_portal'],
+                             iscsi_properties['target_iqn']))
 
         # The /dev/disk/by-path/... node is not always present immediately
         # TODO(justinsb): This retry-with-delay is a pattern, move to utils?
