diff -Naurp nova-2012.1.orig/nova/virt/libvirt/connection.py nova-2012.1/nova/virt/libvirt/connection.py
--- nova-2012.1.orig/nova/virt/libvirt/connection.py	2011-11-10 10:45:37.000000000 -0500
+++ nova-2012.1/nova/virt/libvirt/connection.py	2011-11-15 09:39:46.270362387 -0500
@@ -753,8 +753,8 @@ class LibvirtConnection(driver.ComputeDr
         ajaxterm_cmd = 'sudo netcat - %s' \
                        % get_pty_for_instance(instance['name'])
 
-        cmd = ['%s/tools/ajaxterm/ajaxterm.py' % utils.novadir(),
-               '--command', ajaxterm_cmd, '-t', token, '-p', port]
+        cmd = ['ajaxterm', '--command', ajaxterm_cmd, '-t', token,
+                '-p', port, '-T', '300']
 
         utils.execute(cmd)
         return {'token': token, 'host': host, 'port': port}
