Description: Replace socat with netcat.
Author: Chuck SHort <zulcss@ubuntu.com>
Forwarded: Pending
diff -Naupr nova-2011.3.orig//nova/virt/libvirt/connection.py nova-2011.3//nova/virt/libvirt/connection.py
--- nova-2011.3.orig//nova/virt/libvirt/connection.py	2011-09-23 13:05:47.771063487 -0400
+++ nova-2011.3//nova/virt/libvirt/connection.py	2011-09-23 13:06:15.871063001 -0400
@@ -700,7 +700,7 @@ class LibvirtConnection(driver.ComputeDr
         token = str(uuid.uuid4())
         host = instance['host']
 
-        ajaxterm_cmd = 'sudo socat - %s' \
+        ajaxterm_cmd = 'sudo netcat - %s' \
                        % get_pty_for_instance(instance['name'])
 
         cmd = ['ajaxterm', '--command', ajaxterm_cmd, '-t', token,
