Description: Replace socat with netcat.
Author: Chuck SHort <zulcss@ubuntu.com>
Forwarded: Pending
diff -Naurp nova-2011.3.orig//nova/virt/libvirt/connection.py nova-2011.3//nova/virt/libvirt/connection.py
--- nova-2011.3.orig//nova/virt/libvirt/connection.py	2011-09-22 08:02:23.000000000 -0400
+++ nova-2011.3//nova/virt/libvirt/connection.py	2011-09-23 10:14:23.851185430 -0400
@@ -700,7 +700,7 @@ class LibvirtConnection(driver.ComputeDr
         token = str(uuid.uuid4())
         host = instance['host']
 
-        ajaxterm_cmd = 'sudo socat - %s' \
+        ajaxterm_cmd = 'sudo netcat - %s' \
                        % get_pty_for_instance(instance['name'])
 
         cmd = ['%s/tools/ajaxterm/ajaxterm.py' % utils.novadir(),
